styles.css

```diff
@@ -18,6 +23,21 @@
   --coral: #f28373;
 }
 
+body {
+  background-color: var(--primary-background);
+  color: var(--primary-color);
+  font-family: "Poppins", sans-serif;
+}
+
+main > h1 {
+  font-size: 40px;
+  font-style: normal;
+  font-weight: 400;
+  line-height: 48px;
```

Comment:
Try to remove line-height property. If it does not affect the layout in bad fashion - it should be removed